designcompose/src/testFixtures/java/com/android/designcompose/TestUtils.java

```diff
@@ -39,4 +43,17 @@ public static void triggerLiveUpdate() {
 
         testOnlyTriggerLiveUpdate(actualFigmaToken);
     }
+
+    public static void clearInteractionStates() {
+        InteractionStateManager.INSTANCE.getStates().clear();
```

Comment:
where does `INSTANCE` come from?

Reply:
yeah this is java, not kt. Would you like it converted to kt?

Reply:
When I wrote this there actually wasn't support for Kotlin test fixtures. Looks like it's coming in AGP 8.5: https://b.corp.google.com/issues/259523353?pli=1

Reply:
Oh -- didn't notice. I guess that's fine... I don't know why this file was made in java to begin with -- @timothyfroehlich ?

Reply:
I replied right before you did :)